.vscode/settings.json

```diff
@@ -76,5 +76,6 @@
 	"[typescript]": {
 		"editor.defaultFormatter": "vscode.typescript-language-features"
 	},
-	"typescript.tsc.autoDetect": "off"
+	"typescript.tsc.autoDetect": "off",
+	"debug.javascript.usePreview": false
```

Comment:
please exclude changes to this file in this PR

Reply:
Acknowledged, thank you!